src/main/target/STM32_UNIFIED/target.h

```diff
@@ -159,10 +159,6 @@
 #define USE_ACC
 #define USE_GYRO
 
-#define USE_ACC_MPU6050
```

Comment:
Is there any reason to remove this from STM32F405? Saving ~1000 bytes on a target that has ~ 500 kB of unused flash space seems to be moot, and if we stop building this then it will rot and become unusable in no time at all, so we can just as well delete the code.

Reply:
I2C gyros are no longer viable with native gyro rate sampling. Nobody is going to build a F4 flight controller with I2C gyros. We can add it back only for targets with 1MB of flash, but I don't see the point.

Reply:
As stated, one of the points is to have this code built as part of CI. As seen in #9742, not doing so will mean that sooner or later it will rot and become un-buildable and hard to fix - keeping it built means any rot will be spotted and addressed right when it happens.